### DISCUSSION TITLE
Cannot load just subscriptions without queries nor mutations

### DISCUSSION THREAD

**Discussion by teixeirafrancisca** [MichalLytek/type-graphql#1634] [General]
**Problem**
When running a separate WebSocket (ws) instance from the main application, TypeGraphQL fails to function correctly with the error message "Query must define one or more fields."

**To Reproduce**
Ensure that only subscriptions are loaded into this WebSocket instance, with no queries or mutations.
Attempt to use TypeGraphQL in this configuration, such as defining GraphQL schema with TypeGraphQL decorators.
Observe the error "Query must define one or more fields.

**Expected Behavior**
TypeGraphQL should work correctly even when only subscriptions are loaded. It should not throw errors related to missing query fields when queries and mutations are not expected to be used in this instance.

**Logs**
{
    "message": "Some errors occurred while generating GraphQL schema:
        Type Query must define one or more fields.
        Please check the `details` property of the error to get more detailed info.",
    "name": "Error",
    "stack": "Please check the `details` property of the error to get more detailed info.
        SchemaGenerator.generateFromMetadata (node_modules/type-graphql/build/cjs/schema/schema-generator.js:36:23)
        buildSchema (node_modules/type-graphql/build/cjs/utils/buildSchema.js:29:55)
        buildTypeDefsAndResolvers (node_modules/type-graphql/build/cjs/utils/buildTypeDefsAndResolvers.js:13:56)
  }

**Environment (please complete the following information):**

- OS: [e.g. Windows]
-  Node: 18.12.1
- Package version: type-graphql@"2.0.0-beta.3"
- TypeScript version: 5.3.3

**Additional Context**
The issue occurs specifically when TypeGraphQL is used with a separate WebSocket instance where only subscriptions are loaded. The error suggests that TypeGraphQL expects queries to be defined even though they are not intended to be used in that context. This behavior is unexpected and hinders the use of TypeGraphQL in certain setups where queries and mutations are not applicable.

To overcome the issue we are loading one query and one mutation.

Code and hack to make it work:
```
    const TYPE_GRAPHQL_FILES = path.join(__dirname, './*/types,inputs,subscriptions/**/*.[jt]s');
    const files = glob.sync(TYPE_GRAPHQL_FILES);

    // hack to make type-graphql work with ws instance
    files.push('src/graphql/queries/example-query.ts', 'src/graphql/mutations/example-mutation.ts');

    const loadedResolvers = await loadFiles(typeGraphQLFiles);
    try {
        const { typeDefs, resolvers } = await buildTypeDefsAndResolvers({
            resolvers: loadedResolvers
            validate: false,
        });
        
        ...

    } catch (e) {
        logger.error(e);
    }
```

**Comment by MichalLytek** [comment]
This is `graphql-js` limitation and I can't do anything about it. Just put a dummy query like `currentDate` and live with it.
